0.10/Dockerfile

```diff
@@ -5,11 +5,13 @@ FROM openshift/base-centos7
 
 MAINTAINER Jakub Hadvig <jhadvig@redhat.com>
 
-# Image metadata
-ENV NODEJS_VERSION=0.10 \
-    IMAGE_DESCRIPTION="Node.js 0.10" \
-    IMAGE_TAGS=node,nodejs,nodejs010 \
-    IMAGE_EXPOSE_SERVICES=8080:http
+EXPOSE 8080
+
+ENV NODEJS_VERSION 0.10
```

Comment:
To have a track why, because we want to have an env that user can verify to get information about language/framework version used.